app/models/service.js

```diff
@@ -400,6 +401,29 @@ var Service = Resource.extend({
       return Util.formatSi(this.get('launchConfig.memoryReservation'), 1024, 'iB', 'B');
     }
   }),
+
+  serviceApp: function(){
+    let labels = this.get('launchConfig.labels');
+    let serviceName = this.get('name');
+    let serviceAppUI = labels[C.LABEL.SERVICE_APP_UI] || '';
```

Comment:
This is an odd name for what is essentially the path for the service and would often be `/` so it could just be optional.